tests/components/pvpc_hourly_pricing/test_sensor_logic.py

```diff
@@ -0,0 +1,85 @@
+"""Tests for the pvpc_hourly_pricing component."""
```

Comment:
The test modules should be named after the modules under test. Ie `test_sensor.py`.